src/generators/java/renderers/EnumRenderer.ts

```diff
@@ -34,7 +34,7 @@ ${this.indent(this.renderBlock(content, 2))}
     return `${content};`;
   }
 
-  normalizeKey(value: any): string {
+  normalizeKey(value: unknown): string {
```

Comment:
I do not think that unknown is a good here. I guess any was better. I think that unknown should be used only if you have API entrypoint and the you have unknown user input which is not used by your code at all.
I guess that you should do `any | number | boolean` and maybe bigint if it is possible.

Reply:
as soon as you have `any` what is the reason for adding more types? 🤔